### PR TITLE
update contricution.md

### PR DESCRIPTION
**Pull request by VincyZhang**
## Type of Change

documentation

## Description

update contribution rules
## Expected Behavior & Potential Risk

no

## How has this PR been tested?

spell check

## Dependency Change?

no

### CODE REVIEW

File: docs/contributions.md [hunk 1]
```diff
@@ -1,46 +1,75 @@
...
+
+
+1. How to check third-party dependency license compliance
+	
+    If you are using third-party component by: (1) import lib and uses its API (2) static/dynamic linked (3) copy code and modified (4) other uncertain usage, please contact [Maintainers](inc.maintainers@intel.com) to check license compliance before push code to repo.
+
+
+2. How to apply for repo access
+
+    Create a [ticket](https://opensource.intel.com/jira/servicedesk/customer/portal/1/create/29) to join “intel-extension-for-transformers-write” team.
```
Comment (kevinintel) [inline docs/contributions.md:62]: we should not expose the website

File: docs/contributions.md [hunk 2]
```diff
@@ -1,46 +1,75 @@
...
+1. How to check third-party dependency license compliance
+	
+    If you are using third-party component by: (1) import lib and uses its API (2) static/dynamic linked (3) copy code and modified (4) other uncertain usage, please contact [Maintainers](inc.maintainers@intel.com) to check license compliance before push code to repo.
+
+
+2. How to apply for repo access
+
+    Create a [ticket](https://opensource.intel.com/jira/servicedesk/customer/portal/1/create/29) to join “intel-extension-for-transformers-write” team. 
+
+    if you don't have permission to create ticket, please contact [Maintainers](inc.maintainers@intel.com) to apply for help.
```
Comment (kevinintel) [inline docs/contributions.md:64]: use mailto:inc.maintainers@intel.com like readme in main page

File: docs/CI_introduction.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+|     Unit Test    |   pytest + coverage + gtest |     PASS(No failure, No core dump, No segmentation fault, No coverage drop)         |
+|     Kernel Benchmark          |      Benchmark [Details](../intel_extension_for_transformers/backends/neural_engine/test/kernels/benchmark)               |     PASS(No performance regression)        |
+|     Model Test       |   Pytorch + Tensorflow + Neural Engine + IPEX + CPP Graph   |     PASS(FP32/INT8 No performance regression)         |
+
+# FAQ
+1. How to add examples
+
+    (1) Still need to pass detected CI tests
+
+    (2) Use [Extension Test](https://inteltf-jenk.sh.intel.com/view/nlp-toolkit-validation/job/nlp-toolkit-validation-top-mr-extension/) to cover new examples. If you do not have permission for extension test, please contact [Maintainers](inc.maintainers@intel.com)
```
Comment (kevinintel) [inline docs/CI_introduction.md:23]: use mailto

File: docs/CI_introduction.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+
+2. How to test specific component version
+
+    (1) Use [Extension Test](https://inteltf-jenk.sh.intel.com/view/nlp-toolkit-validation/job/nlp-toolkit-validation-top-mr-extension/) to specify the version. If you do not have permission for extension test, please contact [Maintainers](inc.maintainers@intel.com)
```
Comment (kevinintel) [inline docs/CI_introduction.md:28]: mailto

**Review by kevinintel — COMMENTED** [review]
some comments

**Review by hshen14 — APPROVED** [review]

**Review by kevinintel — APPROVED** [review]
LGTM

**Pull request merged**